tools/scripts/ansible/setup-playbook.yml

```diff
@@ -196,3 +196,5 @@
         --keep --redirect --uir --hsts --staple-ocsp --must-staple
       args:
         creates: /etc/letsencrypt/renewal/{{ inventory_hostname }}.conf
+
+- import_playbook: backup-db-playbook.yml
```

Comment:
Is this a good approach to modularizing and separing of concerns? I partly just wanted to avoid turning setup-playbook.yml into a huge unstructured file. Also it may be convenient to be able to run backup-db-playbook.yml separately. Maybe the same applies to other sections of setup-playbook.yml. What's left there could move to separate playbooks, leaving only "import" lines in the most general playbook?